<ui version="4.0" >
 <class>GenPwDlg</class>
 <widget class="QDialog" name="GenPwDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>460</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>460</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Password Generator</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>10</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="Radio_1" >
        <property name="text" >
         <string>Use follo&amp;wing character groups:</string>
        </property>
        <property name="shortcut" >
         <string>Alt+W</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox1" >
            <property name="text" >
             <string>&amp;Upper Letters</string>
            </property>
            <property name="shortcut" >
             <string>Alt+U</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox2" >
            <property name="text" >
             <string>&amp;Lower Letters</string>
            </property>
            <property name="shortcut" >
             <string>Alt+L</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox3" >
            <property name="text" >
             <string>&amp;Numbers</string>
            </property>
            <property name="shortcut" >
             <string>Alt+N</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox4" >
            <property name="text" >
             <string>&amp;Special Characters</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox5" >
            <property name="text" >
             <string>White &amp;Spaces</string>
            </property>
            <property name="shortcut" >
             <string>Alt+S</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox6" >
            <property name="text" >
             <string>Minus</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox7" >
            <property name="text" >
             <string>U&amp;nderline</string>
            </property>
            <property name="shortcut" >
             <string>Alt+N</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="Radio_2" >
        <property name="text" >
         <string>Use &amp;only following characters:</string>
        </property>
        <property name="shortcut" >
         <string>Alt+O</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="Edit_chars" >
          <property name="maxLength" >
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Length:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Spin_Num" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel5" >
          <property name="text" >
           <string>Quality:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="Progress_Quali" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>1</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum" >
           <number>128</number>
          </property>
          <property name="value" >
           <number>0</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="Check_CollectEntropy" >
          <property name="text" >
           <string>Enable entropy collection</string>
          </property>
          <property name="shortcut" >
           <string>Alt+M</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Check_CollectOncePerSession" >
          <property name="text" >
           <string>Collect only once per session</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel4" >
       <property name="text" >
        <string>New Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Edit_dest" />
     </item>
     <item>
      <widget class="QPushButton" name="ButtonGenerate" >
       <property name="text" >
        <string>Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line3" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="DialogButtons" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>Radio_1</tabstop>
  <tabstop>checkBox1</tabstop>
  <tabstop>checkBox2</tabstop>
  <tabstop>checkBox3</tabstop>
  <tabstop>checkBox4</tabstop>
  <tabstop>checkBox5</tabstop>
  <tabstop>checkBox6</tabstop>
  <tabstop>checkBox7</tabstop>
  <tabstop>Radio_2</tabstop>
  <tabstop>Edit_chars</tabstop>
  <tabstop>Check_CollectEntropy</tabstop>
  <tabstop>Edit_dest</tabstop>
  <tabstop>ButtonGenerate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
